### PR TITLE
bump twoliter to 0.4.3

### PR DESCRIPTION
**Pull request by jmt-lab**
Bump twoliter to 0.4.3

**Testing done:**
Built bottlerocket variant with new version

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by cbgbt — APPROVED** [review]

File: Makefile.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ BUILDSYS_ROOT_DIR = "${CARGO_MAKE_WORKING_DIRECTORY}"
 # For binary installation, this should be a released version (prefixed with a v,
 # for example v0.1.0). For the git sourcecode installation method, this can be
 # any git rev, e.g. a tag, sha, or branch name.
-TWOLITER_VERSION = "v0.4.1"
+TWOLITER_VERSION = "v0.4.2"
```
Comment (cbgbt) [inline Makefile.toml]: This is not the correct version, we should be bumping to 0.4.3.

**Review by cbgbt — CHANGES_REQUESTED** [review]

File: Makefile.toml (hunk 1 above)
Comment (jmt-lab) [reply Makefile.toml]: woops, will fix

**Review by cbgbt — APPROVED** [review]

File: Twoliter.lock [hunk 2]
```diff
@@ -7,11 +7,11 @@ name = "bottlerocket-sdk"
...
-digest = "+Cu8yFgnaFYG5ZQ0eZWRndXxHTLEAR58EhZITbJinLI="
+digest = "myHHKE41h9qfeyR6V6HB0BfiLPwj3QEFLUFy4TXcR10="
 
 [[kit]]
 name = "bottlerocket-core-kit"
 version = "2.0.0"
 vendor = "bottlerocket"
 source = "public.ecr.aws/bottlerocket/bottlerocket-core-kit:v2.0.0"
-digest = "dC/FAJ0K43AugHYTsCeMw/+bjgi1X5zXnpHsbfDnF9Y="
+digest = "vlTsAAbSCzXFZofVmw8pLLkRjnG/y8mtb2QsQBSz1zk="
```
Comment (ginglis13) [inline Twoliter.lock:17]: It's confusing to me that these digests are changing with what appears to be no changes to the kits themselves.

Comment (ginglis13) [reply Twoliter.lock:17]: It appears https://github.com/bottlerocket-os/twoliter/commit/98ca177da88aee9b0d01385c95706e9610154a53 altered how digests are computed

**Review by ginglis13 — APPROVED** [review]

**Pull request merged**